### PR TITLE
Jonas' "overflow: visible" fix, sans core refactor and runtime bbox calc

### PR DESCRIPTION
**Pull request by tsortwehttam**
This is just the slice of Jonas' work from https://github.com/HaikuTeam/mono/pull/707 that we need for release, but omitting the core refactor (which may be risky) and the auto-bbox-calc fix (which may also be risky and which Jonas has identified some additional issues with). Should be OK to merge. Short review.

### CODE REVIEW

File: packages/haiku-serialization/src/bll/Element.js [hunk 1]
```diff
@@ -1588,6 +1588,17 @@ class Element extends BaseModel {
...
+  getParentSvgElement () {
+    let currElem = this
+    while (currElem) {
+      if (currElem.getNameString() === 'svg') {
+        return currElem
+      }
+      currElem = currElem.parent
+    }
+    return null
+  }
```
Comment (stristr) [inline packages/haiku-serialization/src/bll/Element.js]: …I think this is available as `Element.directlySelected.rootSVG` and didn't need to be re-implemented in the `Element` model.

Comment (tsortwehttam) [reply packages/haiku-serialization/src/bll/Element.js]: Matt actually implemented `.rootSVG` on the `HaikuElement` model; this is for `Element`.

Comment (stristr) [reply packages/haiku-serialization/src/bll/Element.js]: Yes, but `Element.directlySelected` uses the `HaikuElement` model.

**Comment by stristr** [comment]
LGTM, merging.

**Pull request merged**